### PR TITLE
added cdn link for Bootstrap 4 and changed the styles for the site's card's h1, p and read more a elements

### PR DESCRIPTION
**Pull request by jcoopnz**

### CODE REVIEW

**Comment by jcoopnz** [comment]
```
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git push
fatal: The current branch bootstrap-format-update has no upstream branch.
To push the current branch and set the remote as upstream, use

    git push --set-upstream origin bootstrap-format-update

Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git push --set-upstream origin bootstrap-format-update
Total 0 (delta 0), reused 0 (delta 0), pack-reused 0
remote: 
remote: Create a pull request for 'bootstrap-format-update' on GitHub by visiting:
remote:      https://github.com/Jordy1311/portfolio_0.1/pull/new/bootstrap-format-update
remote: 
To https://github.com/Jordy1311/portfolio_0.1
 * [new branch]      bootstrap-format-update -> bootstrap-format-update
Branch 'bootstrap-format-update' set up to track remote branch 'bootstrap-format-update' from 'origin'.
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git status
On branch bootstrap-format-update
Your branch is up to date with 'origin/bootstrap-format-update'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   about.html
	modified:   contact.html
	modified:   index.html
	modified:   projects.html
	modified:   style.css

Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git push
Everything up-to-date
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git status
On branch bootstrap-format-update
Your branch is up to date with 'origin/bootstrap-format-update'.

Changes to be committed:
  (use "git restore --staged <file>..." to unstage)
	modified:   about.html
	modified:   contact.html
	modified:   index.html
	modified:   projects.html
	modified:   style.css

Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git commit .
[bootstrap-format-update 600a097] Lines starting
 5 files changed, 14 insertions(+), 10 deletions(-)
 rewrite style.css (100%)
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git status
On branch bootstrap-format-update
Your branch is ahead of 'origin/bootstrap-format-update' by 1 commit.
  (use "git push" to publish your local commits)

nothing to commit, working tree clean
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git pus
git: 'pus' is not a git command. See 'git --help'.

The most similar commands are
	push
	pull
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git push
Enumerating objects: 13, done.
Counting objects: 100% (13/13), done.
Delta compression using up to 4 threads
Compressing objects: 100% (7/7), done.
Writing objects: 100% (7/7), 1.13 KiB | 1.13 MiB/s, done.
Total 7 (delta 5), reused 0 (delta 0), pack-reused 0
remote: Resolving deltas: 100% (5/5), completed with 3 local objects.
To https://github.com/Jordy1311/portfolio_0.1
   50edc48..600a097  bootstrap-format-update -> bootstrap-format-update
Jordans-MacBook-Air:portfolio_0.1 jordancooper$ git status
On branch bootstrap-format-update
Your branch is up to date with 'origin/bootstrap-format-update'.

nothing to commit, working tree clean
Jordans-MacBook-Air:portfolio_0.1 jordancooper$
```

File: projects.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
 <head>
 
 	<title>jordan cooper - projects</title>
+	<link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
```
Comment (suessflorian) [inline projects.html:7]: Question for you; in this case we're referencing the file `https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css`, which ultimately is a file with the name `bootstrap.min.css`. What does the `.min` indicate?

File: index.html [hunk 2]
```diff
@@ -25,26 +26,26 @@
...
 		</p>
 	</div>
 
 	<div class="main-card">
-		<h1>Deffinately code I have not written</h1>
+		<h1 class="make-uppercase">Deffinately code I have not written</h1>
 		<img src="images/proj-img3.jpeg" class="card-img">	
-		<p>
-			Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequatur ipsum nisi corporis! Tempore quas in dolorem, quisquam, voluptatem repellat ullam rem maiores architecto, accusamus cum rerum quaerat odio omnis perferendis! <br><br><a href="#" class="read-more">Read More...</a>
+		<p class="lead">
```
Comment (suessflorian) [inline index.html:47]: Is `lead` a bootstrap class?

File: style.css [hunk 3]
```diff
@@ -1 +1 @@
...
	border-radius: 2px;

	float: right;
	width: 70%;
}
.card-img{
	width: 100%;
	max-width: 530px;
	border-radius: 2px;
}
```
Comment (suessflorian) [inline style.css]: This looks a little funky, might have a look myself to see why we can't make sense of this diff.

**Comment by suessflorian** [comment]
Nice mate! Great to see you start using this `git` workflow on your own. I've got some feedback and we can chat about it face to face when you're ready too as well but the gist is here;

1.  I see you copied and pasted your terminal workflow here. When you're commenting on PR's - you should carry the intention to format things in a nice readable way. Checkout [this link about markdown syntax](https://guides.github.com/pdfs/markdown-cheatsheet-online.pdf).
2. When we create PR's, we leave the merge step to after the review is done.
3. Title of the PR is very long, aim for a more succinct one like "Added bootstrap" and use the description section of your PR to explain what you did like 
```
- Added cdn link to Bootstrap 4
- Changed card's h1, paragraph and read more anchor elements
```

**Comment by suessflorian** [comment]
Just for my feedback above, after reading the markdown syntax guide - you can edit your comment and code block that blob of terminal logs via triple back ticks ` ```<CODE_BLOCK>``` `.

**Pull request merged**